app/views/incidents/upload.html.erb

```diff
@@ -14,7 +14,7 @@
     <% end %>
 
     <p>Use this form to upload multiple incidents at once. The uploaded file
-      must be a JSON file matching our schema <a id="showHideSchema">[show]</a>.
+      must be a JSON or XML file matching our schema <a id="showHideSchema">[show]</a>.
```

Comment:
I would do something on this page to make the valid file formats stand out visually. For example, put "JSON or XML" in bold, or put ("JSON or XML") in the upload button text.